runner/docker/Dockerfile.text_to_speech

```diff
@@ -0,0 +1,21 @@
+ARG BASE_IMAGE=livepeer/ai-runner:base
+FROM ${BASE_IMAGE}
+
+# Install CUDA Toolkit to enable flash attention.
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends cuda-toolkit-12-1 && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN pip install --no-cache-dir \
+    transformers==4.43.3 \
+    peft \
+    deepcache \
+    soundfile \
+    g2p-en \
+    flash_attn \
+    git+https://github.com/huggingface/parler-tts.git@5d0aca9753ab74ded179732f5bd797f7a8c6f8ee
+
+# Copy the app directory into the container
+COPY app /app
```

Comment:
@pschroedl this is already done in the base container right (see https://discord.com/channels/@me/1219448753635725443/1301676937114025995)? If this is not needed due to some errors lets remove it and then this pull request is good to be merged 🚀.

Reply:
@pschroedl stated that this was to prevent having to rebuild the base container during debugging.